bobtemplates/plone_addon/.travis.yml.bob

```diff
@@ -1,28 +1,26 @@
 language: python
+python: 2.7
 sudo: false
 cache:
-  pip: true
   directories:
-    - $HOME/buildout-cache
```

Comment:
no, the changes I made keep the cache also and Travis runs fast as you can see here:

https://travis-ci.org/collective/collective.fingerpointing
https://travis-ci.org/collective/collective.lazysizes
https://travis-ci.org/collective/collective.nitf
https://travis-ci.org/collective/collective.polls
https://travis-ci.org/collective/sc.social.like
https://travis-ci.org/collective/collective.upload

in some cases I also added the `parts/node` directory to the cache also.